src/nvim/ui_compositor.c

```diff
@@ -320,6 +320,15 @@ ScreenGrid *ui_comp_mouse_focus(int row, int col)
       return grid;
     }
   }
+  if (ui_has(kUIMultigrid)) {
+    FOR_ALL_WINDOWS_IN_TAB(wp, curtab) {
+      ScreenGrid *grid = &wp->w_grid_alloc;
+      if (row >= wp->w_winrow && row < wp->w_winrow + grid->rows
+          && col >= wp->w_wincol && col < wp->w_wincol + grid->cols) {
```

Comment:
Does this need to check `grid->mouse_enabled` like the for-loop just above?

Reply:
It probably should, but it does not work currently, some tests are failing. The loop is only run for top level windows, like splits and they don't currently correctly set the `mouse_enabled` status. So, I decided to skip the test.

But now when thinking more about it, I think there might be some cases where it does the wrong thing, if the check is not present. So, I think I will enable it, and ensure that `mouse_enabled` is properly set everywhere it's needed.

Reply:
Fixed, `mouse_enabled` is now set for new `w_grid_alloc` of new windows and for the popupmenu grid to allow this.